backend/api/conferences/types.py

```diff
@@ -254,8 +254,10 @@ def submission_types(self, info) -> List[SubmissionType]:
         return self.submission_types.all()
 
     @strawberry.field(permission_classes=[CanSeeSubmissions])
-    def submissions(self, info) -> Optional[List[Submission]]:
-        return self.submissions.all().select_related(
+    def submissions(
+        self, info, status: Optional[str] = None
```

Comment:
any reason we want to allow the clients to filter them by status?

also does this code work if status is not passed? since it would filter for `status=None` I think?

Reply:
why don't we just return those that are of status "proposed"?

Reply:
I thought this will be more generic. But now we have the voting and the schedule (the admin tool to select them...) the page that is querying the conference's submissions. So think I can remove the filtering and return only the Proposed submissions always